config/locales/hyrax.en.yml

```diff
@@ -1566,6 +1566,21 @@ en:
         resource_type: Pre-defined categories to describe the type of content being uploaded, such as &quot;article&quot; or &quot;dataset.&quot;  More than one type may be selected.
         subject: Headings or index terms describing what the collection is about; these do need to conform to an existing vocabulary.
         title: A name to aid in identifying a collection.
+      pcdm_collection:
```

Comment:
what's the cause of the need for new i18n keys?

Reply:
I don't know where the generation happens, but something generates the part of the i18n key to be `pcdm_collection` instead of `collection`.

Reply:
Without the additional i18n keys, the default keys are used which has `work` in place of `collection` for many of the fields.

Reply:
let's patch it to use the existing `collection` keys, instead of introducing new, duplicate ones

Reply:
The PR was updated to remove the changes to translations.